src/audio/host.c

```diff
@@ -839,8 +839,6 @@ static int host_copy(struct comp_dev *dev)
 
 	tracev_host("cpy");
 
-	if (dev->state != COMP_STATE_ACTIVE)
```

Comment:
I dont see how this improve XRUN or pause release. This commit messages give no detail.